rubygem-foreman_xen/rubygem-foreman_xen.spec

```diff
@@ -82,10 +82,10 @@ GEMFILE
 %exclude %{gem_instdir}/Rakefile
 %exclude %{gem_instdir}/test
 %{gem_spec}
-%{foreman_bundlerd_dir}/%{gem_name}.rb
-%doc %{gem_instdir}/LICENSE
+%{foreman_bundlerd_plugin}
 
 %exclude %{gem_cache}
+%{foreman_assets_plugin}
```

Comment:
Nitpick: please move this up into the main list of files, above the excludes etc.